### PR TITLE
upgrade dropbox vol2

### PR DESCRIPTION
**Pull request by mroz22**
- upgraded dropbox to version 10
- fixed some depracated warnings from express
- added express to integration tests packages. it should be listed there probably too as it is used there.

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/plugins/dropbox.js [hunk 1]
```diff
@@ -90,8 +90,9 @@ class DropboxMock {
             res.end();
         });
 
+        // todo: this endpoint has been deprecated in favor of /search_v2.
         // https://api.dropboxapi.com/2/files/search
-        app.post('/2/files/search', bodyParser.raw(), (req, res) => {
+        app.post('/2/files/search', express.raw(), (req, res) => {
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/plugins/dropbox.js]: rm

File: packages/suite/src/support/tests/npmMocks.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-classes-per-file */
+
+// globally mock npm modules.
+jest.mock('dropbox', () => {
```
Comment (mroz22) [inline packages/suite/src/support/tests/npmMocks.ts:4]: mv

**Review by tomasklim — APPROVED** [review]
LGTM

**Comment by bosomt** [comment]
**QA OK**

app + web version

**Info:**
 - Suite version: web 21.11.0 (55f600be15addfc47aeffc45443137c7c5fef458)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:92.0) Gecko/20100101 Firefox/92.0
 - OS: MacIntel
 - Screen: 1680x1050
 - Device: model 1 1.10.4 regular

**Pull request merged**